monitoring/pra/alerts.yaml

```diff
@@ -0,0 +1,79 @@
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: kafka_connect_src_job
+    type: config
+    value: artesca-data-dr-source-base-queue-connector-metrics
+  - name: kafka_connect_sink_job
+    type: config
+    value: artesca-data-dr-base-queue-connector-metrics
+  - name: dr_sink_instance
+    type: config
+    value: artesca-data-dr
+  - name: RTO
+    type: constant
+    value:  86400 # 24 hours
+  - name: mongo_jobs
+    type: config
+    value: zenko/data-db-mongodb-sharded-shard.*
+
+groups:
+- name: PraAlerts
+  rules:
+
+  - alert: DrResourcePausedWhileOtherRunning
+    expr: |
+      (sum(up{job="${kafka_connect_src_job}", namespace="${namespace}"}) OR vector(0)) != (sum(up{job="${kafka_connect_sink_job}", namespace="${namespace}"}) OR vector(0))
+    for: 1m
+    labels:
+      severity: warning
+    annotations:
+      summary: 'DR Resource Paused While Other Running'
+      description: 'One site s DR resource is paused while the other is not. This could lead to data inconsistency between sites.'
+
+  - alert: DrResourcePausedForTooLong
+    expr: |
+      (sum(up{job="${kafka_connect_src_job}", namespace="${namespace}"}) OR vector(0)) == 0
+    for: 5m
+    labels:
+      severity: warning
+    annotations:
+      summary: 'DR resource paused for too long'
+      description: 'The DR resource has been paused for more than 5 minutes. This could lead to data loss if not addressed promptly.'
+
+  - alert: KafkaConnectOutageSource
+    expr: |
+      sum(rate(kafka_connect_task_error_total_record_errors{job="${kafka_connect_src_job}"}[$__rate_interval])) > 0
+      OR
+      sum(rate(kafka_connect_task_error_total_record_failures{job="${kafka_connect_src_job}"}[$__rate_interval])) > 0
+    for: 1m
+    labels:
+      severity: critical
+    annotations:
+      description: >-
+        Kafka-connect on source is not working nominally. The rate of errors or failures has exceeded 0. This could lead DR to get out of sync if not addressed promptly.
+      summary: 'Kafka Connect not working'
+
+  - alert: KafkaConnectOutageSink
+    expr: |
+      sum(rate(kafka_connect_task_error_total_record_errors{job="${kafka_connect_sink_job}"}[$__rate_interval])) > 0
+      OR
+      sum(rate(kafka_connect_task_error_total_record_failures{job="${kafka_connect_sink_job}"}[$__rate_interval])) > 0
+    for: 1m
+    labels:
+      severity: critical
+    annotations:
+      description: >-
+        Kafka-connect on sink is not working nominally. The rate of errors or failures has exceeded 0. This could lead to data loss if not addressed promptly.
+      summary: 'Kafka Connect not working'
+
+  - alert: WriteTimesLatency
+    expr: |
+      sum(s3_lifecycle_last_timestamp_ms{namespace="${namespace}"}) - sum(mongodb_ss_repl_lastWrite_lastWriteDate{drSinkInstance="${dr_sink_instance}", job="${mongo_jobs}", namespace="${namespace}", cl_role="shardsvr"}) > ${RTO}/2
```

Comment:
should wrap, and no point dividing the threshold by 2

Reply:
also missing filter on `s3_lifecycle_last_timestamp_ms` job

Reply:
https://github.com/scality/Zenko/pull/2130/files#diff-22adcb487a4c13646fe867dd2c3504bedf8e022c2183ff5fbd7b92448fd75038R77-R80